src/components/editArrangementTitle/editArrangementTitle.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { Typography, TextField } from '@material-ui/core'
+
+export class EditArrangementTitle extends Component {
+    constructor (props) {
+        super(props)
+
+        this.handleKeyPress = this.handleKeyPress.bind(this)
+        this.escFunction = this.escFunction.bind(this);
```

Comment:
Can we pass in the name as a prop (in the function in the other file) and have it save the state as the original name here?